documentation/source/newsfragments/README.rst

```diff
@@ -0,0 +1,50 @@
+:orphan:
+
+Release Notes
+=============
+
+We are using `towncrier <https://pypi.org/project/towncrier/>`_ to handle
+our release notes, and this directory contains the input for it -
+"news fragments" which are short files that contain a small
+**ReST**-formatted text that will be added to the next version's
+Release Notes page.
+
+Each file should be named like ``<PULL REQUEST>.<TYPE>.rst``, where
+``<PULL REQUEST>`` is a pull request number, and ``<TYPE>`` is one of:
+
+* ``feature``: New user-facing feature.
+* ``bugfix``: A bug fix.
+* ``doc``: Documentation improvement.
+* ``removal``: Deprecation or removal of public API.
+* ``misc``: A ticket has been closed, but it is not of interest to users.
+
+.. comment_numpy_has_these:
+    * ``new_function``: New user facing functions.
+    * ``deprecation``: Changes existing code to emit a DeprecationWarning.
+    * ``future``: Changes existing code to emit a FutureWarning.
+    * ``expired``: Removal of a deprecated part of the API.
+    * ``compatibility``: A change which requires users to change code and is not
+      backwards compatible. (Not to be used for removal of deprecated features.)
+    * ``c_api``: Changes in the Numpy C-API exported functions
+    * ``new_feature``: New user facing features like ``kwargs``.
+    * ``improvement``: Performance and edge-case changes
+    * ``change``: Other changes
+    * ``highlight``: Adds a highlight bullet point to use as a possibly highlight
+      of the release.
```

Comment:
What's the purpose of this section? Can it be removed?

Reply:
It's just a comment, which is there for people who come back to this and think about adding new categories and need inspiration. We could replace it with a link to the numpy one to make this shorter.

Reply:
Right, it's meant for this discussion mainly - are we okay with the categories that towncrier supports by default, or do we want different ones? For example, the deprecation vs. removal differentiation.